src/App.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Navigation from './components/Navigation';
+import Booklist from './components/booklist';
+import Categories from './components/categories';
+import store from './redux/configureStore';
+
```

Comment:
- [ ] As mentioned by the previous **[reviewer](https://github.com/Becky449/bookstore/pull/5)** to fix your imported files, kindly make sure you did it for all files, as right now  I am getting errors when trying to run the optimized production, all of them are caused by typing errors. And this is why you get the linters errors. Once you fix this you can remove the `/* eslint-disable */` comment
```JavaScript
// Bad
import Navigation from './components/Navigation';
import Booklist from './components/booklist';
import Categories from './components/categories';

// Good
import Navigation from './Components/Navigation';
import Booklist from './Components/booklist';
import Categories from './Components/Categories';
```